Fix `DownloadState.get_ratio()`

drew2a opened the pull request:
This PR fixes #7817 by changing the function to the following:
```python
    def get_all_time_ratio(self) -> float:
        if not self.lt_status or not self.all_time_download:
            return 0
        return self.all_time_upload / self.all_time_download
```

kozlovsky reviewed (changes requested):
I added the comment to the original issue #7817. It looks like the previous formula is correct (except when the `total_done` is zero), and instead of changing the formula, we should display the `all_time_upload` / `all_time_download` counters.

drew2a commented:
@kozlovsky thank you. I've changed the function to the following:

```python
    def get_all_time_ratio(self) -> float:
        if not self.lt_status or not self.all_time_download:
            return 0
        return self.all_time_upload / self.all_time_download
```

Also, I've changed the UI, please check the screenshot in the description for this issue.
